src/docfx/build/toc/ServicePageGenerator.cs

```diff
@@ -71,6 +71,11 @@ internal class ServicePageGenerator
                     var childHref = item.Value.Href.Value;
                     var childUid = item.Value.Uid.Value;
 
+                    if (!string.IsNullOrEmpty(childHref) && childHref.EndsWith('/'))
```

Comment:
check on `item.Value.Href.Value` directly in L71, no need to set `childHref` when `true`?

Reply:
Consider reuse the `GetTocHrefType` method